eventuate-tram-redis-integration-tests/src/test/java/io/eventuate/tram/redis/integrationtests/MessagingTest.java

```diff
@@ -299,16 +305,18 @@ private void assertMessagesConsumed(List<TestSubscription> testSubscriptions, in
     });
   }
 
-  private void waitForRebalance(List<TestSubscription> subscriptions, int totalPartitions) {
+  private void assertSubscriptionPartitions(List<TestSubscription> subscriptions) {
+    assertSubscriptionPartitions(subscriptions, DEFAULT_PARTITION_COUNT);
+  }
+
+  private void assertSubscriptionPartitions(List<TestSubscription> subscriptions, int expectedPartitionCount) {
     Eventually.eventually(EVENTUALLY_CONFIG.iterations,
             EVENTUALLY_CONFIG.timeout,
             EVENTUALLY_CONFIG.timeUnit,
             () -> {
               Assert.assertTrue(subscriptions
```

Comment:
This needs an error message. This is the assert that fails from time to time.

Reply:
done